dashboard/app/coverage_batch.go

```diff
@@ -0,0 +1,147 @@
+// Copyright 2017 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	batch "cloud.google.com/go/batch/apiv1"
```

Comment:
Remove batch package renaming, the package is already named batch.

Reply:
done